pyo3-macros-backend/src/method.rs

```diff
@@ -164,19 +167,15 @@ impl SelfType {
                 } else {
                     syn::Ident::new("extract_pyclass_ref", *span)
                 };
-                error_mode.handle_error(
-                    &py,
-                    quote_spanned! { *span =>
-                        _pyo3::impl_::extract_argument::#method::<#cls>(
-                            #py.from_borrowed_ptr::<_pyo3::PyAny>(#slf),
-                            &mut { _pyo3::impl_::extract_argument::FunctionArgumentHolder::INIT },
-                        )
-                    },
-                )
+                error_mode.handle_error(quote_spanned! { *span =>
+                    _pyo3::impl_::extract_argument::#method::<#cls>(
+                        #py.from_borrowed_ptr::<_pyo3::PyAny>(#slf),
```

Comment:
Below, `py` is used, but here it stays `#py`. Oversight or deliberate?

Reply:
Most likely oversight 🙈